.github/scripts/code_style.sh

```diff
@@ -0,0 +1,63 @@
+#! /bin/bash
+
+# Copyright (c) Microsoft. All rights reserved.
+# Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+set -o errexit # Exit if command failed.
+set -o nounset # Exit if variable not set.
+set -o pipefail # Exit if pipe failed.
+
+# Pass anything as a first parameter to fix code style problems
+FIX=${1:-0}
+
+usage() {
+    echo "${0} [check|fix]" 1>&2
+    exit 1
+}
+
+FIX=${1:-"check"}
+
+# Version 0.67 is the source of truth
+if ! [ -x "$(command -v uncrustify)" ]; then
```

Comment:
Can we create this in temp dir?

 tmp_dir=$(mktemp -d -t uncrustify-XXXX)
 pushd
 cd tmp_dir
 git clone https://github.com/uncrustify/uncrustify.git
 cd uncrustify
 git checkout uncrustify-0.67
 mkdir build
 cd build
 cmake ..
 cmake --build .
 sudo make install
 popd

Reply:
oh didn't know that was a command. using